inc/class-provider.php

```diff
@@ -90,13 +91,24 @@ protected function request( array $args ): MediaList {
 			) );
 		}
 
+		$total = absint( $response->get_headers()['x-wp-total'] );
+		$per_page = absint( $args['per_page'] ?? 40 );
```

Comment:
Do we want to document that `40` is the WordPress default, in case no other value is provided?